debian/startup/kolibri.default

```diff
@@ -32,7 +32,7 @@ fi
 # Will be investigated in 0.8.x when setting a path relative to "~" is
 # supported.
 export KOLIBRI_USER
-export KOLIBRI_LISTEN
+export KOLIBRI_LISTEN_PORT
```

Comment:
just a heads-up that `KOLIBRI_LISTEN_PORT` is deprecated as of 0.10.0. We still it for backward compatibility with old installations, but would be best to migrate to the new `KOLIBRI_HTTP_PORT` if possible

Reply:
I know, this is a fix for `0.9.2`

Reply:
Was `KOLIBRI_HTTP_PORT` introduced in 0.9?

Reply:
nope, this is fine for 0.9.2 - just wanted to make sure you're aware